Testscripts/Linux/dpdkSetup.sh

```diff
@@ -174,7 +174,7 @@ function install_dpdk () {
 	check_exit_status "${1} CONFIG_RTE_LIBRTE_MLX4_PMD=y" "exit"
 	ssh "${1}" "cd $RTE_SDK && make config O=$RTE_TARGET T=$RTE_TARGET"
 	LogMsg "Starting DPDK build make on ${1}"
-	ssh "${1}" "cd $RTE_SDK/$RTE_TARGET && make -j8 && make install"
+	ssh "${1}" "cd $RTE_SDK/$RTE_TARGET && make -j 2>&1 && make install 2>&1"
```

Comment:
is it worth dumping the make output to a file that will be then captured by the log collector?
In case of dpdk build failure at least we'll have the build logs.

Reply:
these outputs will get into dpdkConsoleLogs, that s why I am redirecting error pipe to std.